src/pool.js

```diff
@@ -79,6 +87,18 @@ export default class Pool extends EventEmitter {
       setTimeout(() => { this.emit('finished'); }, 0);
     }
   }
+
+  handleJobAbort(thread, job) {
+    thread.kill();
+
+    const index = this.threads.indexOf(thread);
+    this.threads.splice(index, 1);
+
+    const newThread = spawn(null, [], this.spawnOptions)
+    this.threads.push(newThread)
+
+    this.handleJobDone(newThread, job)
```

Comment:
`handleJobDone()`: Where is that method defined?

Reply:
here: https://github.com/Liu233w/threads.js/blob/c5f067af2d3af7fed50f2c7c51d070f9adb65e93/src/pool.js#L80